README.md

```diff
@@ -10,30 +13,41 @@ Join the community on the [ContinualAI slack](https://join.slack.com/t/continual
 - Install Unix command line tools (like `rm`, and `make`) 
     - On Linux, you already have everything
     - On Mac, install `homebrew` and through homebrew, the `make` package
-    - On Windows... idk, wipe your hard drive and install a proper OS? (Or try Cygwin or Windows Subsystem for Linux)
+    - On Windows, not 100% sure, but try using Powershell, Cygwin or Windows Subsystem for Linux
```

Comment:
no, that's not correct. powershell doesn't prodive unix command line tools afaik

Reply:
Powershell gives `rm`, not sure about `make`.

Reply:
It does not. To get make, you need one of the things I suggested, or grab the binary from gnuwin32 directly. But it's safer to just install cygwin or some other set of unix tools.